### PR TITLE
EDGEV3-1133/refactor-cicd-pipeline-by-using-cypress-docker-image

### PR DESCRIPTION
**Pull request by ololodream**
[EDGEV3-1133]

### CODE REVIEW

File: cypress/utils/actionsByQuesResType.ts [hunk 1]
```diff
@@ -59,9 +59,25 @@ function tick(speedUp: boolean = Cypress.env("fast"), interval: number = DEFAULT
...
   videoBoxPage.getStopButton().click({ force: true }).should("not.exist")
+  cy.wait(1000)
+  cy.get(".response-box .control-btn").contains("Stop").should("not.exist")
   videoBoxPage.getSaveButton().click({ force: true }).should("not.exist")
+
+  cy.wait(10000)
+  cy.wait("@uploadVideo")
+    // .then(() => {
+    //   cy.contains("Uploading...")
+    // })
```
Comment (tonynguyenit18) [inline cypress/utils/actionsByQuesResType.ts:79]: Why we comment this out?

File: cypress/integration/si/retryRecord.js [hunk 2]
```diff
@@ -51,14 +51,14 @@ describe("Retry record video", () => {
...
     arrayOfVideoRetryData.forEach((videoItem, index) => {
       chatBotPage.getRetryButton().eq(videoItem).scrollIntoView().click({ force: true })
       cy.wait(1000)
-
       recordVideo()
-      cy.wait("@uploadVideo")
-      cy.contains("Uploading...").wait(3000).should("not.exist")
+      // cy.wait("@uploadVideo").then(() => {
+      //   cy.contains("Uploading...").wait(3000).should("not.exist")
+      // })
```
Comment (tonynguyenit18) [inline cypress/integration/si/retryRecord.js:61]: Why we comment this out?

**Review by tonynguyenit18 — APPROVED** [review]
Approved but have some questions please have a look.

File: cypress/utils/actionsByQuesResType.ts (hunk 1 above)
Comment (ololodream) [reply cypress/utils/actionsByQuesResType.ts:79]: Good pick up. This function moves to line 82. will delete these dummy lines in next pr.

File: cypress/integration/si/retryRecord.js (hunk 2 above)
Comment (ololodream) [reply cypress/integration/si/retryRecord.js:61]: Good pick up. The uploading check is done inside recordVideo now.

**Pull request merged**